routes/auth.js

```diff
@@ -0,0 +1,76 @@
+const express = require('express');
+const bcrypt = require('bcrypt');
+const createToken = require('../utils/createToken');
+const { models } = require('../models/index');
+
+const routes = express.Router();
+
+routes.post(
+  '/register',
+  // add validation middleware
+  async (req, res, next) => {
+    try {
+      // Check if this name already exist in DB
+      let { name, hotel_id, password } = req.body;
```

Comment:
the documentExists helper would make this endpoint a fraction more concise but barely worth it - you decide